extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/WrapperExtensionContextBuilder.java

```diff
@@ -145,10 +146,10 @@ public static WrapperExtensionContext buildContext(
         var criterionLiteralMapper = new CriterionLiteralMapper();
         var criterionMapper = new CriterionMapper(criterionOperatorMapper, criterionLiteralMapper);
         var edcPropertyUtils = new EdcPropertyUtils();
-        var selfDescriptionService = new SelfDescriptionService(config, monitor);
+        var selfDescriptionService = new SelfDescriptionService(config);
         var ownConnectorEndpointService = new OwnConnectorEndpointServiceImpl(selfDescriptionService);
         var placeholderEndpointService = new PlaceholderEndpointService(
-            config.getString(WrapperExtension.MY_EDC_DATASOURCE_PLACEHOLDER_BASEURL, "http://0.0.0.0")
+            ConfigProps.MY_EDC_DATASOURCE_PLACEHOLDER_BASEURL.getStringOrEmpty(config)
```

Comment:
I wonder whether this kind of small change about the default value may have broken tests in subtle ways...

Reply:
It's from a time where there didn't exist the explicit getStringOrThrow yet

It gets a default, and the default is the EDC_DSP_CALLBACK_ADDRESS which should be a safe bet, right?